packages/vue3/src/components/TextArea/TextArea.vue

```diff
@@ -44,7 +41,7 @@ const model = computed({
   border: 1px solid var(--gray-300);
 }
 
-.wrapper.enabled.focused {
+.wrapper.enabled:has(.textarea:focus) {
```

Comment:
`:has` is still not widely supported, to play it safe I think it's better to avoid using it for now
(reference: https://caniuse.com/?search=%3Ahas)